packages/debugger/app/hooks/use-anonymous-identity.tsx

```diff
@@ -0,0 +1,40 @@
+import type { SignerStateInstance } from "@frames.js/render";
+import type { AnonymousOpenFramesRequest } from "frames.js/anonymous";
+
+type AnonymousSignerInstance = SignerStateInstance<
+  {},
+  AnonymousOpenFramesRequest,
+  {}
+>;
+
+export function useAnonymousIdentity(): AnonymousSignerInstance {
+  return {
+    hasSigner: true,
+    onSignerlessFramePress() {
+      return;
+    },
+    signer: {},
+    async signFrameAction(actionContext) {
+      const searchParams = new URLSearchParams({
+        postType: actionContext.transactionId
+          ? "post"
+          : actionContext.frameButton.action,
+        postUrl: actionContext.frameButton.target ?? "",
```

Comment:
just for the info, this will break the tx button when you confirm the transaction in your wallet. It should be `actionContext.target ?? ""`.